decidim-core/lib/decidim/view_model.rb

```diff
@@ -2,5 +2,13 @@ module Decidim
   class ViewModel < Cell::ViewModel
     include ActionView::Helpers::TranslationHelper
     include ::Cell::Translation
+    include Decidim::ResourceHelper
+    include Decidim::LayoutHelper
+    include Decidim::ApplicationHelper
+    include Decidim::ActionAuthorizationHelper
```

Comment:
I don't think we need all these helpers by default, I'd rather include them manually so we can track the dependencies of each cell

Reply:
But the `Decidim::ResourceHelper` will probably be used for most cards, or _subCells_ such as `CategoryCell`
the rest of them I'll move to `decidim/proposals/view_model`

Reply:
👍